contracts/PriceConverter.sol

```diff
@@ -3,28 +3,20 @@ pragma solidity ^0.8.8;
 
 import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
 
-// TODO: Typing club
 library PriceConverter {
-    //TODO: change priceconverter to a contract type
     function getPrice(
         AggregatorV3Interface priceFeed
     ) internal view returns (uint256) {
         (, int256 answer, , , ) = priceFeed.latestRoundData();
-        return uint256(answer * 10000000000); // TODO: why do we multiply
-        // ANS: The function returns latest price of ETH in terms of USD.
-        // There are 8 decimal places associated with this price feed (there is a "decimals" function in the aggregator contract that says that)
-        // msg.value have 18 decimal places
-        // We want values (price and msg. value) to have same decimal places and same data type (uint256 instead of int256)
```

Comment:
This comment is useful to explain the "magic" number (10000000000), so I'd keep it. Better yet, you can name the magic number to something like `uint256 ETH_USD_MULTIPLIER = 10_000_000_000` and have that comment above it.